libcxx/src/support/runtime/exception_pointer_glibcxx.ipp

```diff
@@ -16,6 +16,17 @@
 // stable ABI), and its rethrow_exception(std::__exception_ptr::exception_ptr)
 // function.
 
+
+#  if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION
```

Comment:
I don't understand why we're modifying this file in this PR. We added a new function to libc++abi, we didn't touch libsupc++ though.

Reply:
Although we didn't change libsupc++, we added some code on the call site which invokes `__cxa_init_primary_exception`.
Since the function is present in libsupc++ i thought it would make sense to support the optimization for libsupc++ as well, and that requires some changes in this file.

I'm totally fine with not doing it in this PR